### ISSUE TITLE
Can't import Device in linux

### ISSUE THREAD

**Issue by Lorac** [picotech/picosdk-python-wrappers#2]
### Setup

* Version of Python: 3.5
* Pico Technology device model: 5244D
* Driver name, version number and platform (32- or 64-bit): 64bit
* Operating system: Linux Debian
* USB port type	(e.g. 2.0): 3.0

### Description

When importing `Device` there is an import error.
```
from ctypes import c_int16, c_int32, c_uint32, c_float, create_string_buffer, byref, cdll, WinDLL
    ImportError: cannot import name 'WinDLL'
```

#### Steps to reproduce the issue

- Create main.py
- Write `from picosdk.device import Device`

#### Actual Result

```
   from ctypes import c_int16, c_int32, c_uint32, c_float, create_string_buffer, byref, cdll, WinDLL
ImportError: cannot import name 'WinDLL'
```

#### Expected Result

No import error.

### Notes

In `__init__.py` from ctypes to import `WinDLL` this condition has to pass : `if _os.name in ("nt", "ce"):` When you are on linux `os.name` returns `posix`.

Maybe change 
```
from ctypes import c_int16, c_int32, c_uint32, c_float, create_string_buffer, byref, cdll, WinDLL
```
to 
```
from ctypes import c_int16, c_int32, c_uint32, c_float, create_string_buffer, byref
if sys.platform == 'win32':
    from ctypes import WinDLL
else:
    from ctypes import cdll
```

After this modification on my side I get with this problem:

```
  File "/home/.../lib/python3.5/site-packages/picosdk/library.py", line 69, in <module>
    from picosdk.device import Device
ImportError: cannot import name 'Device'

```

**Comment by brianbienvenu** [comment]
I've just hit the same issue with a Pico 4444. Will update if I find a workaround.

**Comment by hsmistry** [comment]
@brianbienvenu, the (beta) armhf drivers for BeagleBone Black and Raspberry Pi platforms do not currently support the PicoScope 4444. We hope to address this as soon as possible.

**Comment by hsmistry** [comment]
@Lorac & @brianbienvenu, thank you for raising this issue. I have reported an issue for colleagues to investigate further.

**Comment by Lorac** [comment]
Hello @hsmistry do you have an ETA on a fix?

**Comment by hsmistry** [comment]
Hi @Lorac, I've requested an update from a colleague.

**Comment by hsmistry** [comment]
Hi @Lorac & @brianbienvenu this should be fixed in Changeset e8deec9. Please try the updated the code and confirm if it resolves the issues for your systems.

**Comment by Lorac** [comment]
The specified issue has been fixed thanks!
